license-tool/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+set -e
+set -u
+
+docker build -f ${PWD}/license-tool/Dockerfile -t quay.io/che-incubator/license-tool:next .
```

Comment:
We are not going to push it at this point. So, let's name it like local tag, `nodejs-license-dashes:local`. WDYT?

Reply:
Ok